mlflow/server/js/src/components/RunView.js

```diff
@@ -139,8 +181,25 @@ class RunView extends Component {
             onClose={this.hideRenameRunModal}
             runName={this.props.runName}
             open={this.state.showRunRenameModal} />
+          <DeleteRunModal
+            isOpen={this.state.showDeleteRunModal}
+            onClose={this.onCloseDeleteRunModal}
+            selectedRunIds={[runUuid]} />
+          <RestoreRunModal
+            isOpen={this.state.showRestoreRunModal}
+            onClose={this.onCloseRestoreRunModal}
+            selectedRunIds={[runUuid]}
+          />
         </div>
-
+        {isDeleted ?
```

Comment:
I'd recommend providing a unit test for this conditional rendering.

Reply:
I tried to create a new `RunView.test.js` file to write this unit test. But I am wondering how I am going to prepare the props? I noticed that `Fixtures.createExperiment()` can prepare the `experiment` object quickly, but I am not sure how to prepare other specialized objects like `run`, `params` and `tags`?

Reply:
We are working on adding more test utils... If it does not exist, you can create one. A quick way to get a template up quickly is to console.log some real run data and use them as a start of creating fixtures.